pkg/pillar/types/global.go

```diff
@@ -201,8 +774,8 @@ var GlobalConfigDefaults = GlobalConfig{
 	NetworkSendTimeout: 120,
 
 	UsbAccess:           true, // Contoller likely to default to false
-	SshAccess:           true, // Contoller likely to default to false
-	SshAuthorizedKeys:   "",
+	SSHAccess:           true, // Contoller likely to default to false
+	SSHAuthorizedKeys:   "",
```

Comment:
You can not change these since json will not match when decoding. And since you didn't modify the line yetus will not complain.